packages/docs-examples/components/scrollbar/scrollbar-input-customization/scrollbar-input-customization-example.ts

```diff
@@ -0,0 +1,61 @@
+import { Component, ViewEncapsulation } from '@angular/core';
+import { KbqScrollbarModule, KbqScrollbarOptions } from '@koobiq/components/scrollbar';
+
+/**
+ * @title scrollbar-input-customization-example
+ */
+@Component({
+    standalone: true,
+    selector: 'scrollbar-input-customization-example',
+    imports: [KbqScrollbarModule],
+    template: `
+        <div
+            class="layout-row"
+            [style.gap.px]="16"
+        >
+            <div
+                kbq-scrollbar
+                style="height: 300px; max-width: 300px; overflow: auto"
```

Comment:
```suggestion
                style="height: 300px; max-width: 300px;"
```
библиотека переопределяет на `hidden`, зачем `auto`?

Reply:
Да, работает и без `overflow: auto` :) удалил